example/omega1s1-wip.red

```diff
@@ -17,18 +17,35 @@ let loopn (n : int) : Path s1 base base =
   | pos n ⇒
     elim n [
     | zero ⇒ auto
-    | suc (n ⇒ loopn) ⇒ trans s1 (λ i → loop i) loopn
+    | suc (n ⇒ loopn) ⇒ trans s1 loopn (λ i → loop i)
```

Comment:
This line and the similar line below seem to be causing the trouble.

Reply:
Heh.. So the issue is that when it associates in one way it is fast, and in the other way infeasible? That's very interesting.

Reply:
It somehow makes sense. I changed the order to make the lemma easier to prove. The new one produces `isuc` from the outside and is perfect for induction.

Reply:
Shall we try a harder version of the lemma? I guess analogous to tail recursive function

Reply:
This performance asymmetry also exists in cubicaltt, btw.

Reply:
I guess to prove the "fast" version, it will take some gnarly coercion. Sadly, the goals are tens of thousands of lines long, so it's a little hard to tell what needs to be done.

Reply:
@favonia Can you elaborate on why it makes sense? I guess the performance of coe in fcom depends more on the complexity of the tubes than that of the cap?

Reply:
@ecavallo I am not sure. I have to think harder.